zuul.d/architecture-jobs.yaml

```diff
@@ -53,6 +53,7 @@
       cifmw_openshift_setup_skip_internal_registry_tls_verify: true
       cifmw_networking_env_def_file: "networking-env-definition.yml"
       cifmw_networking_mapper_networking_env_def_path: "/etc/ci/env/networking-environment-definition.yml"
+      cifmw_architecture_automation_file: "{{ cifmw_architecture_repo }}/automation/vars/hci.yaml"
```

Comment:
err... nope? the reproducer is supposed to build the filename based on `cifmw_architecture_scenario` ... ?

Reply:
yes, but this is for the crc VA job which doesn't use the architecture role

Reply:
oh. ok, "reproducer" role you mean. Indeed, that parameter is injected by the `configure_controller.yml` nested in the reproducer.

Maybe use the `cifmw_architecture_scenario` instead of hard-coding the `hci.yaml` ? That's my only concern here. Or move that specific parameter to `cifmw-architecture-validate-hci` job definition ?

Reply:
fair enough, I though I added it to the child job which is hci specific, let me make that more general

Reply:
updated  the PR to use `cifmw_architecture_scenario`